### PR TITLE
PV/PVC changes added

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/74 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Test A -  Ran Cert-CSI Automation run
- [x] Test B - Ran unit tests and integration tests

### CODE REVIEW

File: service/controller.go [hunk 1]
```diff
@@ -1645,7 +1659,80 @@ func addMetaData(params map[string]string) map[string]string {
...
+			PublishedNodeIds: exportList,
+			VolumeCondition: &csi.VolumeCondition{
+				Abnormal: abnormal,
+				Message:  message,
+			},
+		},
+	}, nil
+}
+
+func (s *service) getVolumeCondition(ctx context.Context, isiConfig *IsilonClusterConfig) (bool, string) {
```
Comment (ashish2207) [inline service/controller.go:1724]: can we renamed this method to something like isVolumeAbnormal, as when we are returning false, we are saying volume is healthy.

**Review by ashish2207 — APPROVED** [review]

**Comment by mjsdell** [comment]
@Sakshi-dell have the latest updates mentioned in csi-general been reflected here yet?

**Comment by Sakshi-dell** [comment]
> @Sakshi-dell have the latest updates mentioned in csi-general been reflected here yet?

Updated PR with the latest discussion

**Review by prablr79 — APPROVED** [review]

**Comment by prablr79** [comment]
@Sakshi-dell  can you resolve the conflicts here ? so that we can merge this PR ?

**Pull request merged**